Doc/library/contextlib.rst

```diff
@@ -236,10 +236,11 @@ Functions and classes provided:
 
    For example, the output of :func:`help` normally is sent to *sys.stdout*.
    You can capture that output in a string by redirecting the output to an
-   :class:`io.StringIO` object::
+   :class:`io.StringIO` object. For convenience, the new stream is returned
```

Comment:
It's not necessarily a *new* stream.

Reply:
Hrm, I was meaning new in the sense of now being the one used for `stdout`. Perhaps "replacement" would be clear?

Reply:
Sorry I missed your reaction earlier in the year. I've now made this change (in 14d7692706f7e98cd9f3dfdfa60ab329f6892886).